<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #e4ecfa;</string>
  </property>
  <widget class="QLineEdit" name="le_id">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>29</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff;
border-radius: 8px;
color: #000000;
padding-right: 3px;
padding-left: 3px;
padding-bottom: 1px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_pw">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ffffff;
border-radius: 8px;
color: #000000;
padding-right: 3px;
padding-left: 3px;
padding-bottom: 1px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: #275efe;
	border-radius: 10px;
	color: #ffffff;
}
QPushButton:hover {
	background-color: #274fff;
}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_signin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: none;
	background-color: #2f3545;
	border-radius: 10px;
	color: #ffffff;
}
QPushButton:hover {
	background-color: #49526b;
	color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans KR</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>PW</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
